orgbee/src/app/pages/freedom-wall/freedom-wall.component.html

```diff
@@ -1 +1,223 @@
-<p>freedom-wall works!</p>
+<div class="freedomwall-page">
+  <div class="top">
+    <div class="header">
+      <app-header 
+        [title]="'Computer Society Freedom Wall'" 
+        [orgName]="'Welcome to'" 
+        [showImage]="false" 
+        [height]="'short-height'"></app-header>
+    </div>
+  
+    <div class="button-container">
+      <button [disabled]="showFilterMessage" class="create-post-btn" (click)="toggleModal()">
```

Comment:
In creating elements, set the order to:
```
<element
#templateRefereceVariable
attribute=directive
*structural=directive
attribute=value
[property]=binding
(event)=binding
[(twoWay)]=binding>
</element>
```

See all the other elements in this file, to follow this sequence. See the link for reference. 
https://docs.google.com/presentation/d/1Mdcq3WcHksBlbCvxi6wx5vajn4v8LGlJRnBIu3x4fvw/edit#slide=id.g2ce365fc0ac_0_0